src/primitives/mod.rs

```diff
@@ -26,6 +26,11 @@ pub enum Bech32 {}
 #[derive(Copy, Clone, PartialEq, Eq, PartialOrd, Ord, Hash)]
 pub enum Bech32m {}
 
+/// The codex32 checksum algorithm, defined in [BIP-93].
+/// [BIP-350]: <https://github.com/bitcoin/bips/blob/master/bip-0093.mediawiki>
+#[derive(Copy, Clone, PartialEq, Eq, PartialOrd, Ord, Hash)]
+pub enum Codex32 {}
```

Comment:
In 4da0a7b9784a7fe92db39a7d645d4bee97999d4d:

This can live in a doccomment or something but we probably shouldn't export it from this library.

Reply:
Done.